samples/msgext-ai-sentiment-analysis/nodejs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "msgext-ai-sentiment-analysis",
+    "version": "1.0.0",
+    "msteams": {
+        "teamsAppId": null
+    },
+    "description": "Sentiment analysis using message extension and Azure Open AI",
```

Comment:
Sentiment analysis of Teams chat messages using message extension and Azure Open AI